.github/workflows/coderabbit.yaml

```
@@ -0,0 +1,42 @@
+
+language: ko-KR
```

actionlint (1.7.7)

unexpected key "language" for "workflow" section. expected one of "concurrency", "defaults", "env", "jobs", "name", "on", "permissions", "run-name"

(syntax-check)